### ISSUE TITLE
Unable to install other packages when this package install in React 18

### ISSUE THREAD

**Issue by sandeshthapa07** [keppelen/react-facebook-login#353]
When I try to install other package in my project when this package is installed  it throws and error saying this package is deprecated to react 18 try installing in React 16 version

**Comment by anhdungadg** [comment]
Yep, me too! It's work very well on React 16. But it's got error when I'm upgrade to React 17. Maybe the version 18 are the same!

**Comment by sandeshthapa07** [comment]
you can another package **https://www.npmjs.com/package/react-facebook**  for login.it worked for me

**Comment by anhdungadg** [comment]
I'm done with this command on React 17.
`npm install react-facebook --legacy-peer-deps`
